dubbo-rpc/dubbo-rpc-triple/src/main/java/org/apache/dubbo/rpc/protocol/tri/call/AbstractServerCall.java

```diff
@@ -173,12 +173,14 @@ public final void onComplete() {
     }
 
     @Override
-    public final void onMessage(byte[] message) {
+    public final void onMessage(TripleFlowControlFrame data) {
```

Comment:
`FlowControlFrame` should not be known by `Call` class

Reply:
change to object